cfgov/cfgov/urls.py

```diff
@@ -74,243 +81,229 @@ def empty_200_response(request, *args, **kwargs):
-    url(r'^blog/(?P<path>.*)$',
-        RedirectView.as_view(
-            url='/about-us/blog/%(path)s', permanent=True)),
-    url(r'^newsroom/(?P<path>.*)$',
-        RedirectView.as_view(
-            url='/about-us/newsroom/%(path)s', permanent=True)),
+    re_path(r'^parents/(?P<path>.*)$', RedirectView.as_view(
+        url='/money-as-you-grow/%(path)s', permanent=True)),
+    re_path(r'^blog/(?P<path>.*)$', RedirectView.as_view(
+        url='/about-us/blog/%(path)s', permanent=True)),
+    re_path(r'^newsroom/(?P<path>.*)$', RedirectView.as_view(
+        url='/about-us/newsroom/%(path)s', permanent=True)),
 
-    url(r'^the-bureau/(?P<path>.*)$',
-            RedirectView.as_view(url='/about-us/the-bureau/%(path)s',
-                                 permanent=True)
-    ),
-    url(r'^about-us/leadership-calendar/(?P<path>.*)$', RedirectView.as_view(
-        url='/about-us/the-bureau/leadership-calendar/%(path)s',
-        permanent=True)),
+    re_path(r'^the-bureau/(?P<path>.*)$', RedirectView.as_view(
+        url='/about-us/the-bureau/%(path)s', permanent=True)),
 
-    url(r'^doing-business-with-us/(?P<path>.*)$',
+    re_path(r'^about-us/leadership-calendar/(?P<path>.*)$',
         RedirectView.as_view(
-            url='/about-us/doing-business-with-us/%(path)s', permanent=True)),
+            url='/about-us/the-bureau/leadership-calendar/%(path)s',
+            permanent=True)),
+
+    re_path(r'^doing-business-with-us/(?P<path>.*)$', RedirectView.as_view(
+        url='/about-us/doing-business-with-us/%(path)s', permanent=True)),
 
-    url(r'^external-site/$', ExternalURLNoticeView.as_view(),
+    re_path(r'^external-site/$', ExternalURLNoticeView.as_view(),
         name='external-site'),
 
-    url(r'^subscriptions/new/$', govdelivery_subscribe, name='govdelivery'),
+    re_path(r'^subscriptions/new/$', govdelivery_subscribe,
+        name='govdelivery'),
 
-    url(r'^govdelivery-subscribe/', include([
-        url(r'^success/$',
-            TemplateView.as_view(
-                template_name='govdelivery-subscribe/success/index.html'),
+    re_path(r'^govdelivery-subscribe/', include([
+        re_path(r'^success/$', TemplateView.as_view(
+            template_name='govdelivery-subscribe/success/index.html'),
             name='success'),
-        url(r'^error/$',
-            TemplateView.as_view(
-                template_name='govdelivery-subscribe/error/index.html'),
+        re_path(r'^error/$', TemplateView.as_view(
+            template_name='govdelivery-subscribe/error/index.html'),
             name='user_error'),
-        url(r'^server-error/$',
-            TemplateView.as_view(
-                template_name='govdelivery-subscribe/server-error/index.html'),
+        re_path(r'^server-error/$', TemplateView.as_view(
+            template_name='govdelivery-subscribe/server-error/index.html'),
             name='server_error')],
         namespace='govdelivery')),
```

Comment:
This is a hairy one... If we make the list of paths for `include()` into a two-tuple, this should work:

```python
    re_path(r'^govdelivery-subscribe/',
            include(
                (
                    [
                        re_path(r'^success/$', TemplateView.as_view(
                            template_name='govdelivery-subscribe/success/index.html'),
                            name='success'),
                        re_path(r'^error/$', TemplateView.as_view(
                            template_name='govdelivery-subscribe/error/index.html'),
                            name='user_error'),
                        re_path(r'^server-error/$', TemplateView.as_view(
                            template_name='govdelivery-subscribe/server-error/index.html'),
                            name='server_error')
                    ],
                    'govdelivery'
                ),
                namespace='govdelivery')
            ),
```

(I exploded this a little bit to emphasize the syntax with whitespace.)

Reply:
Updated `govdelivery-subscribe` regular expression to a two-tuple with an application namespace